virtio-devices/src/vhost_user/vu_common_ctrl.rs

```diff
@@ -32,6 +32,7 @@ pub struct VhostUserConfig {
 pub struct VhostUserHandle {
     vu: Master,
     ready: bool,
+    migration_support: bool,
```

Comment:
nit: `migratable` or `supports_migration` ?

Reply:
Migratable has a special meaning with the trait, so I'll pick `supports_migration` :)